phoonnx/phonemizers/base.py

```
@@ -190,7 +190,7 @@ def __init__(self, form: Literal["NFC", "NFD", "NFKC", "NFKD"] = "NFD"):
 
     def phonemize_string(self, text: str, lang: str) -> str:
```

Ruff (0.15.0)

[warning] Unused method argument: `lang`

(ARG002)